### PR TITLE
Webapp in intellij support the new telemetry schema

### PR DESCRIPTION
**Pull request by zmssp**
Send the telemetry use the new telemetry wrapper, the old telemetry still needs for one or two release.
This commit mainly covers the webapp telemetry in intellij.

### CODE REVIEW

File: PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/runner/AzureRunProfileState.java [hunk 1]
```diff
@@ -59,20 +60,40 @@ public ExecutionResult execute(Executor executor, @NotNull ProgramRunner program
...
+                sendOperError(ErrorType.systemError, err.getMessage(), telemetryMap);
+                sendOperEnd();
                 this.onFail(err.getMessage(), processHandler);
                 this.sendTelemetry(telemetryMap, false, err.getMessage());
             });
         return new DefaultExecutionResult(consoleView, processHandler);
     }
 
+    protected void sendOperStart(){
+    }
```
Comment (Flanker32) [inline PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/runner/AzureRunProfileState.java]: make it abstract?

Comment (zmssp) [reply PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/runner/AzureRunProfileState.java]: If make it abstract, several classes will have compile error.
This commit just for webapp.
After fix all productions, then I will make it a abstract method

File: PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/util/TelemetryUtil.java [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+    public static void sendTelemetryOpError(ErrorType errorType, String errMsg, Map<String, String> properties) {
+        TelemetryManager.getInstance().getProducer().sendError(errorType, errMsg, properties, null);
+    }
+
+    public static void sendTelemetryInfo(Map<String, String> properties) {
+        TelemetryManager.getInstance().getProducer().sendInfo(properties, null);
+    }
+
+    public static Map<String, String> buildWebappProperties(Map<String, String> properties,
```
Comment (Flanker32) [inline PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/util/TelemetryUtil.java]: For me, i prefer to make this as a method of webAppSettingModel, like getTelemetryProperties and return a map

File: PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/util/TelemetryUtil.java [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+            result.put(TelemetryConstants.SUBSCRIPTIONID, webAppSettingModel.getSubscriptionId());
+            result.put(TelemetryConstants.CREATE_NEWWEBAPP, String.valueOf(webAppSettingModel.isCreatingNew()));
+            result.put(TelemetryConstants.CREATE_NEWASP, String.valueOf(webAppSettingModel.isCreatingAppServicePlan()));
+            result.put(TelemetryConstants.CREATE_NEWRG, String.valueOf(webAppSettingModel.isCreatingResGrp()));
+            result.put(TelemetryConstants.FILETYPE, MavenRunTaskUtil.getFileType(webAppSettingModel.getTargetName()));
+        } catch (Exception ignore) {
+        }
+        return result;
+    }
+}
```
Comment (Flanker32) [inline PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/util/TelemetryUtil.java]: please create an empty line in the end

**Pull request closed without merging**